src/homeWork14/List.java

```diff
@@ -0,0 +1,65 @@
+package homeWork14;
+
+import java.util.ArrayList;
+
+
+public class List {
+
+    public static int countOccurrence(ArrayList<String> list, String target) {
+        int count = 0;
+
+        for (String str : list) {
+            if (str.equals(target)) {
+                count++;
+            }
+        }
+        return count;
+    }
+
+    public static ArrayList<Integer> toList(int[] array) {
+        ArrayList<Integer> list = new ArrayList<>();
+
+        for (int j : array) {
```

Comment:
rename `j`  to something more meaningful. `number', `value`

Reply:
+++